app/_src/gateway/production/sizing-guidelines.md

```diff
@@ -169,6 +170,17 @@ the better choice, as it provides near-unlimited throughput capacity. In this
 scenario, adding more cache memory would not increase maximum throughput by
 much.
 
+{% if_version gte:3.5.x %}
+When {{site.base_gateway}} is operating in `hybrid` mode with a large number of configuration
+entities (routes, services etc.), it can benefit from the "Dedicated Configuration Processing" option.
```

Comment:
```suggestion
{% if_version gte:3.5.x %}
When {{site.base_gateway}} is operating in `hybrid` mode with a large number of configuration
entities (routes, services, etc.), it can benefit from the **dedicated configuration processing** option.
```
I wouldn't capitalize this, it's not a proper noun. If we want emphasis on the feature name, better to bold or italicize it.